scripts/install/deploy/install_db.sh

```diff
@@ -28,3 +28,39 @@ fi
 
 sudo cp timeless_pg.service /lib/systemd/system/
 sudo systemctl start timeless_pg.service
+
+#checks if the database exists and create if not
+if [! psql -lqt | cut -d \| -f 1 | grep -qw "timelessdb"; ]; then
+    echo "Creating database: timelessdb"
+    sudo -u postgres psql -c "CREATE DATABASE timelessdb;"
+fi
+
+#checks if the user exists and create if not
+if [! psql -t -c '\du' | cut -d \| -f 1 | grep -qw "timeless_user"; ]; then
+    echo "Creating user: timeless_user"
+    sudo -u postgres psql -c "CREATE USER timeless_user WITH
+        SUPERUSER
+        CREATEDB
+        CREATEROLE
+        INHERIT
+        LOGIN
+        ENCRYPTED PASSWORD 'timeless_pwd';"
+fi
+
+company = $(sudo -u postgres -H -- psql -d timelessdb -c "INSERT INTO company
+(name, code, address) values ('Timeless', 'Tm', '')")
+role = $(sudo -u postgres -H -- psql -d timelessdb -c "INSERT INTO role
+(name, works_on_shifts, company_id) values ('Administrator', False, $company)")
+password = "pass from rultor"
+sudo -u postgres -H -- psql -d timelessdb -c "INSERT INTO employee
+    (first_name, last_name, username, phone_number, birth_date,
+    registration_date, account_status, user_status, email, password, pin_code,
+    comment, company_id, role_id) values
+    ('First', 'Last', 'timeless', '988888', '', '', 'active', 'active',
+    'abc@xyz.com', $password, '10', 'Timeless user', $company, $role
```

Comment:
@paulodamaso why mock data such as ``abc@xyz.com``?

Reply:
@amihaiemil I don't have this data, I'll ask to @emilianodellacasa

Reply:
@emilianodellacasa What values should we put here on this user account?

Reply:
@paulodamaso Sorry, I lost your comments among the others: for this sensitive data, please use:


credentials.account.admin.first_name
credentials.account.admin.last_name
credentials.account.admin.username
credentials.account.admin.email
credentials.account.admin.password
credentials.account.admin.pincode